system/MenuButton.module.scss

```diff
@@ -0,0 +1,37 @@
+.body {
+  display: inline-block;
+  padding: 40px;
+  color: inherit;
+}
+
+.hamburger {
+  display: block;
+  cursor: pointer;
+  z-index: var(--z-index-high);
+
+  &.active .bar:nth-child(2) {
+    opacity: 0;
+  }
+
+  &.active .bar:nth-child(1) {
+    transform: translateY(16px) rotate(45deg);
+  }
+
+  &.active .bar:nth-child(3) {
+    transform: translateY(-16px) rotate(-45deg);
+  }
+}
+
+.bar {
+  display: block;
+  width: 32px;
+  height: 8px;
+  margin: 8px auto;
+  background: inherit;
+  background-color: currentColor;
+  color: inherit;
+
+  // (@xBalbinus) These timings should line up with the color transitioning speed in ThemeManager.tsx
+  transition: transform 0.3s ease-in-out, opacity 0.3s ease-in-out, color 0.3s ease-in-out;
+  -webkit-transition: transform 0.3s ease-in-out, opacity 0.3s ease-in-out, color 0.3s ease-in-out;
```

Comment:
is there any way to just share this code if it's supposed to be in sync?

at the very least, you can use an scss var to avoid duplication here.

manually-keep-in-sync tech debt is really nice to avoid